lib/workers/package/versions.js

```diff
@@ -122,6 +124,8 @@ function determineUpgrades(npmDep, config) {
           allUpgrades[upgradeKey].isMajor = true;
         } else if (type === 'minor') {
           allUpgrades[upgradeKey].isMinor = true;
```

Comment:
Interestingly, we don't seem to use `isMinor` anywhere in the code yet. But it's good to add `isPatch` anyway for consistency.